build-system/tasks/e2e/build-bento-e2e.js

```diff
@@ -0,0 +1,132 @@
+const {compileCss} = require('../css');
+const {
+  buildBinaries,
+  buildExtensions,
+  buildNpmCss,
+} = require('../extension-helpers');
+
+const fastGlob = require('fast-glob');
+const {endBuildStep} = require('../helpers');
+const path = require('path');
+const {existsSync, mkdirSync, writeFile} = require('fs-extra');
+
+/**
+ * Build the Bento end to end page
+ *
+ * @param {!Object} options
+ * @return {!Promise<void>}
+ */
+async function buildBentoE2E(options) {
+  const e2eFixtures = await fastGlob(
+    'extensions/*/1.0/test-e2e/e2e-fixture.js'
+  );
+
+  const isMissingCss =
+    (await fastGlob(`src/bento/components/*/1.0/dist/styles.css`)).length === 0;
+  if (isMissingCss) {
+    await compileCss(options);
+    await buildExtensions({...options, compileOnlyCss: true});
+  }
```

Comment:
When would the css files be missing? Because `dist` wasn't called before? If that's the case, let's look for a more generalized approach we can take here that asks the question "was `dist` called before". Otherwise we're risking weird build-skew issues where only the CSS files are being built

Reply:
That was one of the problems, css files previously were not being saved when dist was ran and moved to the temp workspace. I think I did fix that, so maybe this code is no longer needed.